### DISCUSSION TITLE
Introduction: Ecofo (Tom & Kyle)

### DISCUSSION THREAD

**Discussion by Tomlyons1** [life-itself/community#1047] [Introductions]
Hi everyone, 

I’ve been following along with the conversations on the Life Itself YouTube channel for some time now. I felt called to reach out to tell you about a project that my friend Kyle and I have been working on called Ecofo. We feel it would be a great fit for your online community.

To give you a sense of what Ecofo is, it’s custom video software for online communities engaging in dialogue practice. The intention is to facilitate small-group dialogue more sustainably, creating a container for community members to regularly practice relating to the Sacredness of one another. We believe this is the strongest foundation that a community can have. 

So far, we’re working alongside [Emergent Commons](https://www.youtube.com/watch?v=BFcHlpTYG94&ab_channel=TheStoa) and [Voicecraft](https://www.voicecraft.io/), and we’d love to include Life Itself because of how closely aligned our visions are. Our core intention is to strengthen the communication within and between communities in the broader emerging ecosystem. So far we've hosted over 150 conversations. 

Here’s a [short trailer](https://youtu.be/eVTkobvoU7c) we made with Emergent Commons.

-------------------------

If anyone is interested in trying this, we'd love to do some experimental sessions to give you a visceral sense of the experience. The sessions are always between 4 people and last 90 minutes. Each session is centered around a question for inquiry that's selected by the group before the session.

This [video](https://youtu.be/oE2ZcEOAo1s) will give you a short overview of everything involved in the experience.

Let me know if you'd like to try it out and we can organize some sessions. 
Looking forward to hearing from you,
Tom.

**Comment by Tomlyons1** [comment]
@laurenwigmore @nathenf, I'm in New Zealand so the community calls are a bit early for me to come along and introduce myself. But If you have time, perhaps we could organize an Ecofo session between you two, myself, and Kyle? 

I know time is precious, so if this isn't of interest to you right now, I could also send an open invitation in the LI Whatsapp to see if anyone else wants to do a session and report back on how they find it. 

Let me know your thoughts (:

**Comment by rufuspollock** [comment]
@Tomlyons1 welcome! Understand the community calls hour is not so great (we are thinking of moving it back which may work better).

Thank-you for sharing your project and work - really interesting.

> If anyone is interested in trying this, we'd love to do some experimental sessions to give you a visceral sense of the experience. The sessions are always between 4 people and last 90 minutes. Each session is centered around a question for inquiry that's selected by the group before the session.

Understand better now what you are offering and seeking to do, so thank-you 🙏

This is something we can flag on our research collective meeting and ask about. You could also post on the events channel and offer this and see if folks were interested.

/cc-ing John Oliver @baouroux here as he is also interested in these things.

**Comment by Tomlyons1** [comment]
Thank you @rufuspollock 🙏 I'll make a post in the events channel.

John, let me know if you're interested and we can organize a session.
